src/azure-cli/azure/cli/command_modules/container/custom.py

```diff
@@ -562,14 +560,14 @@ def container_logs(cmd, resource_group_name, name, container_name=None, follow=F
 
 
 def container_export(cmd, resource_group_name, name, file):
+    api_version = "2021-03-01"
```

Comment:
Why do you hard code  `"api_version = 2021-03-01"`here

Reply:
If you need to use the fixed api-version, could you refer to the design documents [multi-api-aware-modules](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#multi-api-aware-modules) to implement it, so that it is easier to manage and maintain the api-version used by `ContainerInstanceManagementClient`?

Reply:
@zhoxing-ms  I previously hard coded the API because the Track2 SDK removed the property `container_group_client.api_version` so we had no way to get the API version from the SDK.

I have added the API version to the Profile like specified in the design documents. Tests pass, but the linter is giving me the error:

`No module named 'azure.mgmt.containerinstance.v2021_03_01'`

Any ideas why the API would be used in the version of the container instance Management SDK? Looks like this is something in `cli-core`.

Full trace: 
`No module named 'azure.mgmt.containerinstance.v2021_03_01'
Traceback (most recent call last):
  File "c:\users\josephporter\source\repos\azure-cli\env\lib\site-packages\knack\cli.py", line 231, in invoke
    cmd_result = self.invocation.execute(args)
  File "c:\users\josephporter\source\repos\azure-cli\env\lib\site-packages\knack\invocation.py", line 223, in execute
    cmd_result = parsed_args.func(params)
  File "c:\users\josephporter\source\repos\azure-cli\env\lib\site-packages\knack\commands.py", line 146, in __call__
    return self.handler(*args, **kwargs)
  File "c:\users\josephporter\source\repos\azure-cli\env\lib\site-packages\knack\commands.py", line 253, in _command_handler
    result = op(client, **command_args) if client else op(**command_args)
  File "c:\users\josephporter\source\repos\azure-cli\env\lib\site-packages\azdev\operations\linter\__init__.py", line 118, in run_linter
    create_invoker_and_load_cmds_and_args(az_cli)
  File "c:\users\josephporter\source\repos\azure-cli\src\azure-cli-core\azure\cli\core\file_util.py", line 74, in create_invoker_and_load_cmds_and_args
    invoker.commands_loader.load_arguments()
  File "c:\users\josephporter\source\repos\azure-cli\src\azure-cli-core\azure\cli\core\__init__.py", line 512, in load_arguments
    cmd.load_arguments()  # this loads the arguments via reflection
  File "c:\users\josephporter\source\repos\azure-cli\src\azure-cli-core\azure\cli\core\commands\__init__.py", line 313, in load_arguments
    super(AzCliCommand, self).load_arguments()
  File "c:\users\josephporter\source\repos\azure-cli\env\lib\site-packages\knack\commands.py", line 104, in load_arguments
    cmd_args = self.arguments_loader()
  File "c:\users\josephporter\source\repos\azure-cli\src\azure-cli-core\azure\cli\core\commands\command_operation.py", line 125, in arguments_loader
    op = self.get_op_handler(self.op_path)
  File "c:\users\josephporter\source\repos\azure-cli\src\azure-cli-core\azure\cli\core\commands\command_operation.py", line 59, in get_op_handler
    handler = import_module(mod_to_import)
  File "C:\Users\josephporter\AppData\Local\Programs\Python\Python38\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'azure.mgmt.containerinstance.v2021_03_01'`

Reply:
Actually, `azure.mgmt.containerinstance` is not a multi-api package, so it does not need to be defined like this.

Reply:
@lmazuel Hi, due to the api_version in the client from Python SDK is no longer a public member variable, do you know how to get the api_version defined by the client from Python SDK in track 2?
Such as: [code link](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/containerinstance/azure-mgmt-containerinstance/azure/mgmt/containerinstance/_configuration.py#L51)

Reply:
@zhoxing-ms Correct, it is not a multi-api package. I have reverted all my previous changes for multi-api support.

I can get the API version from `container_group_client._config.api_version`, but that of course leads to style errors for accessing a protected member:
`W0212: Access to a protected member _config of a client class (protected-access)`

I looked through the SDK again and don't see any other way to get the API version. For now, I've changed it to use the `_config` while we look for alternatives.

Reply:
Maybe we can add `# pylint: disable=protected-access` to disable this linter check as an alternative that is not recommended first.
@lmazuel Do you have any suggestions?

Reply:
@zhoxing-ms I have added the line to suppress the warning. Looks like all the checks are passing now.

Since we do not have any alternatives yet, should this be an acceptable workaround and mean the PR is ready to complete?

Reply:
OK